spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/actuate/AbstractGatewayControllerEndpoint.java

```diff
@@ -61,6 +71,8 @@ public class AbstractGatewayControllerEndpoint implements ApplicationEventPublis
 
 	private static final Log log = LogFactory.getLog(GatewayControllerEndpoint.class);
 
+	private static final String ENDPOINT_PREFIX = "/actuator/gateway";
```

Comment:
Actuator endpoints won't necessarily be at `/actuator`. What happens if they are configured to be at a different endpoint?

Reply:
Should used that as a default, but use https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/server/ManagementServerProperties.java#L55 if set.